documentation/modules/rn-2.2.adoc

```diff
@@ -94,3 +94,12 @@ The error status message for a VM with no operating system on the *Migration pla
 .Network, storage, and VM referenced by name in the Plan CR are not displayed in the web console.
 
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
+
+.A log archive file downloaded from the {project-short} web console might include logs related to a deleted resource
```

Comment:
This looks like a copy/paste directly from the bug. Please rewrite according to our style, for example, briefer title.